int8 STATE_ERROR=0
int8 STATE_OFF=1
int8 STATE_INITIALIZING=2
int8 STATE_STANDBY=3
int8 STATE_OPERATIONAL=4

Header header
int8 state
string info
string active_mode
string[] standby_modes